contracts/test/proposals/vips/vip16.sol

```diff
@@ -705,5 +752,26 @@ contract vip16 is Proposal {
             address(migratorRouter.OLD_VOLT()),
             addresses.mainnet("V1_VOLT")
         );
+
+        /// compound bad debt sentinel
+        assertTrue(
+            badDebtSentinel.isCompoundPcvDeposit(
+                addresses.mainnet("PCV_DEPOSIT_MORPHO_USDC")
+            )
+        );
+        assertTrue(
+            badDebtSentinel.isCompoundPcvDeposit(
+                addresses.mainnet("PCV_DEPOSIT_MORPHO_DAI")
+            )
+        );
+        assertEq(
+            badDebtSentinel.pcvGuardian(),
+            addresses.mainnet("PCV_GUARDIAN")
+        );
+        assertEq(
+            badDebtSentinel.comptroller(),
+            addresses.mainnet("COMPTROLLER")
```

Comment:
maybe let's be more specific in the name, this is Compound V2's comptroller

Reply:
done